packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -2908,6 +2909,9 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll Custom HOTP Factor
+Enrollment via the Authentication API is currently not supported for Custom HOTP Factor.  Please refer to the Factors API documentation [here](/docs/reference/api/factors/#enroll-custom-hotp-factor) if you would like to enroll users for this type of factor.
+
```

Comment:
Regarding these changes about HOTP factor, is this a release-specific change?

Reply:
Yes, it should be included in our Release Notes.